src/test/java/emissary/server/api/EmissaryApiTest.java

```diff
@@ -178,6 +178,25 @@ void version() {
         }
     }
 
+     @Test
+     void versionSimple() {
+     GitRepositoryState mockRepoState = mock(GitRepositoryState.class);
```

Comment:
getting build failures with not being able to find GitRepositoryState... need to make constructor public in GitRepositoryState.java